src/scales/scale.linear.js

```diff
@@ -31,7 +31,7 @@ export default class LinearScale extends LinearScaleBase {
 
   // Utils
   getPixelForValue(value) {
-    return this.getPixelForDecimal((value - this._startValue) / this._valueRange);
+    return value === null ? NaN : this.getPixelForDecimal((value - this._startValue) / this._valueRange);
```

Comment:
For example bubble controller checks only for `null` now, this would mean null is not skipped?

Reply:
This is the pixel value being returned, not the data value. We're using `null` for the data value

Reply:
ah, sorry I missed the reference to bubble controller initially. fixed